checkov/main.py

```diff
@@ -653,6 +655,10 @@ def add_parser_args(parser: ArgumentParser) -> None:
                help="exclude extra resources (resources without violations) from report output",
                action='store_true',
                env_var='CKV_SKIP_RESOURCES_WITHOUT_VIOLATIONS')
+    parser.add('--deep-analysis',
+               default=False,
+               action='store_true',
+               help='Enable combine tf graph and rf plan graph')
```

Comment:
I don't think we should add it yet, we shouldn't expose the flag before the logic is working

Reply:
so for now we can just put something hardcoded?

Reply:
yes